### DISCUSSION TITLE
Batch processing feature?

### DISCUSSION THREAD

**Discussion by slt-uxd** [nagadomi/waifu2x#436] [Ideas]
Hello - I just did a test comparing standard upsampling (used XnConvert), Gigapixel, and Waifu, and Waifu had the best result. Very impressive! 

I'd love a batch processing feature. Any plans to have one? I'd be happy to pay for it (depending on the cost of course). Keep up the great work!

**Comment by nagadomi** [comment]
Is it about the web service?
Some local applications support batch conversion of image folders.

[waifu2x-caffe](https://github.com/lltcggie/waifu2x-caffe/releases) (windows, NVIDIA)
[waifu2x-ncnn-vulkan](https://github.com/nihui/waifu2x-ncnn-vulkan) (Windows/Linux/MacOS, Intel / AMD / Nvidia / Apple-Silicon)
[nunif/waifu2x](https://github.com/nagadomi/nunif/tree/master/waifu2x) (my python implementation of waifu2x)

**Reply by slt-uxd** [reply]
Yes but I just realized that it makes more sense to have the batch processing feature as part of a local application so thank you for those links!
